backend/editor/api.py

```diff
@@ -188,6 +189,25 @@ async def findFooter(response: Response):
 
 # Post methods
 
+@app.post("/nodes")
+async def createNode(request: Request):
+    """
+    Creating a new node in a taxonomy
+    """
+    incomingData = await request.json()
+    id = incomingData["id"]
+    main_language = incomingData["main_language"]
+    if (id == None):
+        raise ValueError("Invalid id: %s", id)
+    if (main_language == None):
+        raise ValueError("Invalid main language: %s", main_language)

```

Comment:
You should raise [HTTPError](https://fastapi.tiangolo.com/tutorial/handling-errors/#use-httpexception) with [400 code](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#client_error_responses)